docs/sources/administration/_index.md

```diff
@@ -1,10 +1,12 @@
 ---
 cascade:
-  labels:
-    products:
-      - cloud
-      - enterprise
-      - oss
+  - _target:
+      path: /docs/grafana/*/administration/**
+    labels:
+      products:
+        - cloud
+        - enterprise
```

Comment:
Does this mean that all docs under Admin receive the Enterprise label? This should not be the case.